api/v1alpha1/reference_types.go

```diff
@@ -27,12 +26,22 @@ type ObjectReference struct {
 
 type ResourceReference struct {
 	// +required
-	v3alpha1.ElementMeta `json:",inline"`
+	ElementMeta `json:",inline"`
 
 	// +optional
 	ReferencePath []ocmmetav1.Identity `json:"referencePath,omitempty"`
 }
 
+type ElementMeta struct {
```

Comment:
Why did you copy this out? 🤔

Reply:
Because having the `Version` field on resources is required but it's crazy to expect users to specify a verison for every resource 😆

Reply:
We could try to change it upstream?

Reply:
Meh. :D It's fine. :D